HW3/task_2.py

```diff
@@ -0,0 +1,9 @@
+age = 27
+my_name = "Eugen"
+age_in_month = age*12
```

Comment:
style nit: use spaces before and after the operation symbol for more readability
```suggestion
age_in_month = age * 12
```

Reply:
got it